challenges/deploy-WebV/README.md

```diff
@@ -0,0 +1,74 @@
+# Deploy WebValidate
+
+## Background
+
```

Comment:
I wouldn't use "our"

Focus should be on the consistent load as we're not running any kind of load testing.

There's no mention of the logs / metrics / dashboards which is the "why"

Reply:
added a couple sentences to clarify